Sources/Workspace/InitPackage.swift

```diff
@@ -219,6 +268,36 @@ public final class InitPackage {
                                 path: "Sources"),
                         ]
                     """
+                } else if packageType == .macro {
+                    param += """
+                            // Macro implementation, only built for the host and never part of a client program.
+                            .macro(name: "\(pkgname)Macros",
+                                   dependencies: [
+                                     .product(name: "SwiftSyntax", package: "swift-syntax"),
+                                     .product(name: "SwiftSyntaxMacros", package: "swift-syntax"),
+                                     .product(name: "SwiftOperators", package: "swift-syntax"),
+                                     .product(name: "SwiftParser", package: "swift-syntax"),
+                                     .product(name: "SwiftParserDiagnostics", package: "swift-syntax"),
+                                     .product(name: "SwiftCompilerPlugin", package: "swift-syntax"),
```

Comment:
I think we can simplify this since `SwiftSyntaxMacros` implies most of these. I had success with just two dependencies, `SwiftSyntaxMacros` and `SwiftCompilerPlugin`.